docs/_blog/liana-payjoin.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Refreshing timelocked UTXOs with Payjoin"
+description: "Using Payjoin to Preserve Privacy and Lower Fees in Timelocked Wallets"
+date: "2025-04-09"
+authors:
+  - Armin Sabouri
+tags:
+  - privacy
+  - integrations
+---
+
+The highest form of self-custody involves using multiple geographically distributed hardware keys. But what happens if you lose a quorum of those keys? Many users turn to paid services like [Casa](https://casa.io/) or [Unchained](https://unchained.com/) to have a third party manage a subset of their keys in a self-custodial way. [Liana](https://wizardsardine.com/liana/) addresses this recovery problem differently. It allows users to opt into a spending policy that includes a time-locked recovery path.
```

Comment:
This is a very in-the-know bitcoiner orientation. I'd prefer if it were oriented around the benefit for inheritence specifically and did not mention Casa and Unchained.